api/Makefile

```diff
@@ -29,7 +29,6 @@ install:
 	# ETC
 	$(INSTALL_DIR) $(ETC_INSTALLDIR)/api
 	$(INSTALL_RW_DIR) $(ETC_INSTALLDIR)/api/configuration
-	$(INSTALL_RW_DIR) $(ETC_INSTALLDIR)/api/configuration/ssl
 	$(INSTALL_RW_DIR) $(ETC_INSTALLDIR)/api/configuration/security
```

Comment:
Is there any reason to maintain the `api` directory? After moving the certs, only empty directories remain, we could remove these now.

Reply:
The RBAC security.yaml configuration file is still there

Reply:
I thought we had a `security.yaml` file there but that configuration is now in `wazuh-server.yml` so I have deleted it from the Makefile 👍🏼